includes/class-syndication-encryption.php

```diff
@@ -48,8 +49,8 @@ public static function set_encryptor( $encryptor ) {
 	 *
 	 * @return false|string
 	 */
-	public static function encrypt( $data ) {
-		$encryptor = self::get_encryptor();
+	public function encrypt( $data ) {
+		$encryptor = $this->get_encryptor();
 		return $encryptor->encrypt( $data );
```

Comment:
Why call a public method when we can just grab the property directly?

```suggestion
		return $this->encryptor->encrypt( $data );
```